packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -751,14 +751,7 @@ const TaxResidenceField = ({
     );
 };
 
-const TaxIdentificationNumberField = ({
-    is_tin_popover_open,
-    setIsTinPopoverOpen,
-    setIsTaxResidencePopoverOpen,
-    disabled,
-    required = false,
-}) => {
-    const { isDesktop } = useDevice();
+const TaxIdentificationNumberField = ({ disabled, required = false }) => {
```

Comment:
Since this is temporary, can we handle it using a prop? Something like hide_popover? This way we needn't recreate it

Reply:
But we always have version and git history of file so we can recover this anytime and we don't know when we will enable it again 😁